javascript/ecmascript-2015/string-flexibility/tagged-template-literals.md

```diff
@@ -44,43 +44,41 @@ aspects:
 ---
 ## Content
 
-ES6 introduces a new feature called template strings that make it easier to work with strings by adding string interpolation and multi-line strings:
+ES6 introduced a new feature called template strings that make it easier to work with strings by adding string interpolation and multi-line strings:
 
-```
+```js
 let company = "Enki";
 console.log(`Hello ${company}!`);
 //prints: "Hello Enki!"
 ```
 
-Template literals are denoted by the O&#769; character, rather than `''` or `""`. When you use template strings, you can have multiline strings:
-
-```
+Template literals are denoted by the `` ` `` character, rather than `''` or `""`. When you use template strings, you can have multiline strings:
 
+```js
 let aMultilineString = `
 Hello
 I am a multiline string
-`
-
+`;
 ```
 
 As well as embedding _any_ JavaScript expression inside of a pair of curly braces prefixed by a dollar sign: `${5 + 5}`. The expression will be cast to a string, then replaced inside the template string.
```

Comment:
Instead of "As well as..." we can say:

> Beyond variables, template literals actually allow us to interpolate _any_ JavaScript expression, for example `${5 + 5}`. The expression will be cast to a string, then replaced inside the template string.

Reply:
done